sound/soc/sdw_utils/soc_sdw_utils.h

```diff
@@ -12,6 +12,7 @@
 #include <sound/soc-acpi.h>
 
 #define SOF_SDW_MAX_DAI_NUM		8
+#define RT1316_MAX_BQ_REG		580
```

Comment:
This is pretty RT specific feels like it should be in some RT header rather than the common util header.